worker/error-handler/package.json

```diff
@@ -14,11 +14,12 @@
     "pino-pretty": "^9.1.1"
   },
   "devDependencies": {
-    "eslint": "^7.29.0"
+    "eslint": "^7.32.0",
+    "@babel/eslint-parser": "^7.19.1",
```

Comment:
this should not be required and can be removed

Reply:
done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)